src/components/Icon/Icon.tsx

```diff
@@ -71,7 +71,7 @@ export const Icon: React.ForwardRefExoticComponent<IconProps & React.RefAttribut
         } else if (isComponentSvgData(data)) {
             ({viewBox} = data.defaultProps);
         } else if (isSvgrData(data)) {
-            const el = data({}) as React.ReactElement;
+            const el = React.createElement(data, {});
```

Comment:
As you can see in line 125, we don't want the default props to be applied. It was intentional

Reply:
Yes, I understand.
But on line 61, you would like to get the width and height from the viewBox in case they were not specified.
And the viewBox is passed to defaultProps in svgr transformed icons.
sorry, if I miss something

Reply:
Also, I can try a alternative solution- extends available  types for component svg data 
```
export function isComponentSvgData(data: SVGIconData): data is SVGIconComponentData {
    return (typeof data === 'object' || typeof data === 'function') && 'defaultProps' in data;
}
```